example/src/main/kotlin/com.expedia.graphql.sample/validation/DataFetcherExecutionValidator.kt

```diff
@@ -0,0 +1,29 @@
+package com.expedia.graphql.sample.validation
+
+import com.expedia.graphql.sample.exceptions.ConstraintError
+import com.expedia.graphql.sample.exceptions.ValidationException
+import com.expedia.graphql.schema.Parameter
+import com.expedia.graphql.schema.hooks.DataFetcherExecutionPredicate
+import graphql.schema.DataFetchingEnvironment
+import javax.validation.ConstraintViolation
+import javax.validation.Valid
+import javax.validation.Validator
+
+class DataFetcherExecutionValidator(private val validator: Validator?) : DataFetcherExecutionPredicate() {
```

Comment:
since you define the class to be initialized with validator -> why not make it non nullable?